pkg/core/block/block.go

```diff
@@ -127,13 +141,30 @@ func (b *Block) Trim() ([]byte, error) {
 // Serializable interface.
 func (b *Block) DecodeBinary(br *io.BinReader) {
 	b.Base.DecodeBinary(br)
-	br.ReadArray(&b.Transactions)
+	maxContentsPerBlock := br.ReadVarUint()
```

Comment:
Why not just `contentsPerBlock` (`contentsCount`)?

Reply:
Replaced it with contentsCount, thank you.